langchain4j-cdi-core/pom.xml

```diff
@@ -0,0 +1,28 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>dev.langchain4j.cdi</groupId>
+        <artifactId>langchain4j-cdi-parent</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <artifactId>langchain4j-cdi-core</artifactId>
```

Comment:
We agreed at the 2025-06-06 meeting that having the `langchain4j-cdi` everywhere is the best plan. This avoids duplicating the name used in LangChain4J itself.